Android/Shared/src/main/java/com/bentley/sample/shared/ActivityTimer.kt

```diff
@@ -88,20 +89,20 @@ class ActivityTimer(private val nameTitle: String = "ACTIVITY") {
         for (i in maxLengths.indices) {
             maxLengths[i] = headerRow[i].length
         }
-        val jsonCheckpoints = JsonArray()
+        val jsonCheckpoints = JSONArray()
         for (checkpoint in checkpoints) {
             val row = arrayOf(
                 checkpoint.first,
                 timeFormat.format(checkpoint.second),
                 timeDeltaString(lastTime, checkpoint.second),
                 timeDeltaString(startTime, checkpoint.second),
             )
-            jsonCheckpoints.add(jsonOf(mapOf(
+            jsonCheckpoints.put(jsonOf(
                 "action" to row[0],
                 "timestamp" to checkpoint.second.time.epochMillisToISO8601(),
                 "step" to timeDelta(lastTime, checkpoint.second),
                 "total" to timeDelta(startTime, checkpoint.second),
-            )))
+            ).value)
```

Comment:
This should probably be `objectValue` instead of value, since we know the value is an object. Possibly better would be to construct a `JSONObject` from the map instead of our `JSONValue` since we don't need that extra wrapper here.

Reply:
Switched to JSONObject(mapOf(...